docs/devel/e2e-node-tests.md

```diff
@@ -34,147 +34,180 @@ Documentation for other releases can be found at
 ```
 
-### Running the tests
+This will:
+- Build the Kubernetes source code
+- Create a new GCE instance using the default test image
+  - Instance will be called *test-e2e-node-containervm-v20160321-image*
+- Lookup the instance public ip address
+- Copy a compressed archive file to the host containing the following binaries:
+  - ginkgo
+  - kubelet
+  - kube-apiserver
+  - e2e_node.test (this binary contains the actual tests to be run)
+- Unzip the archive to a directory under /tmp/gcloud*
+- Run the tests using the `ginkgo` command
+  - Starts etcd, kube-apiserver, kubelet
+  - The ginkgo command is used because this supports more features than running the test binary directly
+- Output the remote test results to STDOUT
+- `scp` the log files back to the local host under /tmp/_artifacts/e2e-node-containervm-v20160321-image
+- Delete the created GCE instance
 
-1. If running tests against a running host on gce
+# Additional Remote Options
 
-  * Make sure host names are resolvable to ssh by running `gcloud compute config-ssh` to
-    update ~/.ssh/config with the GCE hosts.  After running this command, check the hostnames
-    in the ~/.ssh/config file and verify you have the correct access by running `ssh <host>`.
+## Run tests using different images
 
-  * Copy [template.properties](../../test/e2e_node/jenkins/template.properties)
+This is useful if you want to run tests against a host using a different OS distro or container runtime than provided by the default image.
+
+List the available test images using gcloud.
+
+```sh
+gcloud compute images list --project kubernetes-node-e2e-images | grep "e2e-node-"
+```
 
-    * Fill in `GCE_HOSTS` with the name of the host
+This will output a list of the available images.
 
-  * Run `test/e2e_node/jenkins/e2e-node-jenkins.sh <path to properties file>`
-    * **Must be run from kubernetes root**
+Then run:
+
+```sh
+make test_e2e_node REMOTE=true IMAGES="<comma-separated-list-images>"
+```
+
+## Run tests against a running GCE instance (not an image)
+
+This is useful if you have an host instance running already and want to run the tests there instead of on a new instance.
+
+```sh
+make test_e2e_node REMOTE=true HOSTS="<comma-separated-list-of-hostnames>"
+```
 
-2. If running against a host anywhere else
+## Keep instance around after tests run
 
-  * **Requires password-less ssh and sudo access**
+This is useful if you want to inspect or debug the host after the tests complete, or if you want to run
+additional tests without recreating a new instance each time the tests are run.
+**Note: When not-deleting an instance, subsequent tests run using the same image will *reuse the existing host*,
+even if they would normally provision a new instance.**
 
-    * Make sure this works - e.g. `ssh <hostname> -- sudo echo "ok"`
-    * If ssh flags are required (e.g. `-i`), they can be used and passed to the
-tests with `--ssh-options`
+```sh
+make test_e2e_node REMOTE=true DELETE_INSTANCES=false
```

Comment:
Did you mean just the DELETE_INSTANCES=false part, or both that and REMOTE=true?